strongbox-data-service/src/main/resources/db/changelog/v1.0.0/v1.0.0.2__ArtifactEntry.xml

```diff
@@ -27,6 +27,7 @@
             <o:property name="artifactPath" type="string" />
             <o:property name="sizeInBytes" type="long" />
             <o:property name="downloadCount" type="integer" />
+            <o:property name="created" type="date" />
```

Comment:
Should I rename it to v1.0.0.3__ArtifactEntry.xml and apply previous changes or add changeSet tag in the same file? I am asking because I didn't find an appropriate example in commits history and I don't have enough experience in liquibase.

Reply:
Please create new changelog file called `v1.0.0.9__ArtifactEntry.xml` (next possible minor version) and follow previous files convention.